compiling java LCIO

Hello,

 I'm trying to create a new LCIO jar file for  this project https://atlaswww.hep.anl.gov/asc/jas4pp/  using the master version (2.17). When I use maven to build the java part of LCIO, I'm getting errors:

```
SIOLCParameters.java:[99,22] cannot find symbol
[ERROR] symbol:   method getDoubleKeys()
[ERROR] location: variable params of type hep.lcio.event.LCParameters
```

Is any place to download pre-built jar for most recent lcio library ?

best, Sergei

--- a/src/aid/EVENT/LCParameters.aid
+++ b/src/aid/EVENT/LCParameters.aid
@@ -108,6 +108,10 @@ public interface LCParameters
     /** Returns a list of all keys of string parameters.
      */
     public const StringVec& getStringKeys() const ;
+
+   /** Returns a list of all keys of string parameters.
+    */
+    public const StringVec& getDoubleKeys() const ;
 @endif
 
 

